.mvn/.gradle-enterprise/gradle-enterprise-workspace-id

```diff
@@ -0,0 +1 @@
+juftkio6ovccraxoz7vxuvtuba
```

Comment:
What is this ID used for? Should it be private?

Reply:
According [here](https://docs.gradle.com/enterprise/maven-extension/):

> Since version 1.2, the Gradle Enterprise Maven extension captures an identifier used to uniquely represent a given workspace.
>    For versions 1.2.3+, this identifier is stored under .mvn/.gradle-enterprise/gradle-enterprise-workspace-id. The .mvn/.gradle-enterprise folder should NOT be committed under version control.

I'd remove it

Reply:
Wait, let's make instead make `.gitignore` a badass one, I'll push a new one

Reply:
pushed, no idea why is not updating here, however you can see it in the "commit" tab